waku/waku_archive/archive.nim

```diff
@@ -111,6 +117,33 @@ proc handleMessage*(
   let insertDuration = getTime().toUnixFloat() - insertStartTime
   waku_archive_insert_duration_seconds.observe(insertDuration)
 
+proc syncMessageIngress*(
+    self: WakuArchive,
+    msgHash: WakuMessageHash,
+    pubsubTopic: PubsubTopic,
+    msg: WakuMessage,
+) {.async.} =
+  let insertStartTime = getTime().toUnixFloat()
+
+  (await self.driver.put(msgHash, pubsubTopic, msg)).isOkOr:
+    waku_archive_errors.inc(labelValues = [insertFailure])
+    trace "failed to insert message",
```

Comment:
shouldn't we log the failures at `debug` or `error` level? or it's something that happens a lot?

Reply:
yeah potentially a lot of messages so I went with trace.